src/resources/js/tickets-block.js

```diff
@@ -555,8 +561,10 @@ tribe.tickets.block  = {
 			}
 		);
 
-		// Repeat every 15 seconds
-		setTimeout( obj.checkAvailability, 15000 );
+		// Repeat every 60 (filterable via tribe_tickets_availability_check_interval ) seconds
```

Comment:
Thought here, I almost think this repeat should happen after the `$.post` finishes (whether it's successful or errors).